examples/util/lib/ellipsis.dart

```diff
@@ -1,2 +1,2 @@
 dynamic blockEllipsis; // Use as replace="/=. blockEllipsis;/{ ... }/g"
-dynamic ellipsis; // Use as replace="/ellipsis;/.../g"
+dynamic ellipsis; // Use as replace="/ellipsis;?/.../g"
```

Comment:
Why does `ellipsis` have the `?` but `blockEllipsis` doesn't?

Do we use blockEllipsis anywhere?

Reply:
I've removed all current uses of `blockEllipsis`, but left it there in case we might need it later.